pkgs/misc/anarchism/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "anarchism-${version}";
+  version = "15.0";
+  src = fetchurl {
+    url = https://0xacab.org/ju/afaq/repository/master/archive.tar.gz;
+    sha256 = "b081885152628c67c0f6d9101be17095d84684bc9dd2c7d4e50f5744614883ef";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/doc/anarchism/html
+    mkdir $out/share/doc/anarchism/markdown
+    mkdir -p $out/share/applications/
+    cp -r html $out/share/doc/anarchism/
+    cp -r markdown $out/share/doc/anarchism/
+    echo \[Desktop Entry\] > $out/share/applications/anarchism.desktop
+    echo 'Encoding=UTF-8' >> $out/share/applications/anarchism.desktop
+    echo 'Name=An Anarchist FAQ' >> $out/share/applications/anarchism.desktop
+    echo 'GenericName=Anarchism' >> $out/share/applications/anarchism.desktop
+    echo 'Comment=Exhaustive exploration of Anarchist theory and practice' >> $out/share/applications/anarchism.desktop
+    echo Exec=xdg-open file:///$out/share/doc/anarchism/html/index.html >> $out/share/applications/anarchism.desktop
+    echo 'Terminal=false' >> $out/share/applications/anarchism.desktop
+    echo 'Type=Application' >> $out/share/applications/anarchism.desktop
+    echo 'Categories=Education;' >> $out/share/applications/anarchism.desktop
+    echo 'StartupNotify=true' >> $out/share/applications/anarchism.desktop
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Exhaustive exploration of Anarchist theory and practice.";
+    longDescription = ''
+      The Anarchist FAQ is an excellent source of information regarding
+      Anarchist (libertarian socialist) theory and practice. It covers all major
+      topics, from the basics of Anarchism to very specific discussions of
+      politics, social organization, and economics.
+    '';
+    homepage = http://www.anarchistfaq.org;
+    license = stdenv.lib.licenses.gpl2;
```

Comment:
I don't see any notice on AFAQ site that it is under GPLv2.

Reply:
Both can be applied: "Permission is granted to copy, distribute and/or modify this document under the terms of the GNU Free Documentation License, Version 1.1 or any later version published by the Free Software Foundation, and/or the terms of the GNU General Public License, Version 2.0 or any later version published by the Free Software Foundation." http://www.anarchism.pageabode.com/afaq/intro.html